.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Please check if the PR fulfills these requirements:
+
+* [ ] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
```

Comment:
This manual check is not necessary anymore.

Reply:
I personally use that link to make sure I am listing the right keywords in `[<TYPE>:<MODULE>]`, but if you think it is no longer necessary, we can remove it. If remove it here, should we remove it in the main Submitty repo as well, for consistency?

Reply:
We now have a github action for the "PR Title Check"
Yes, we can remove it from the submitty repo too.